scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,25 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
```

Comment:
readabilit, and it's "safer"... (we should probably change this throughout the file, but ok)

for example, what if you want to do this: `"${foo}bar"`
if you want to use parameter expansion operations, you have to use the curly braces (`${filename%.*}` - removes the extension)
...
what you start using arrays (ok, ash doesn't support them, I know), you have to use the curlies...

I just use them always, makes things more readable, but here they aren't really needed